server/src/main/java/info/doseamigos/meds/MedService.java

```diff
@@ -15,20 +16,20 @@
      * @param name The Name of the med to look up.
      * @return A new Med object for the user with the name.
      */
-    Med addByName(String name);
+    Med addByName(AuthUser authUser, String name);
```

Comment:
I need to update the java docs for all these methods.  The auth user is passed into the service for validation reasons (can this auth user actually see/modify the meds for the amigo listed?)